sdk/js/src/token_bridge/getIsTransferCompleted.ts

```diff
@@ -64,6 +72,74 @@ export async function getIsTransferCompletedTerra(
   return false;
 }
 
+/**
+ * Return if the VAA has been redeemed or not
+ * @param tokenBridgeAddress The Injective token bridge contract address
+ * @param signedVAA The VAA that is being checked for redemption
+ * @param walletPKHash Wallet information
+ * @returns true if the VAA has been redeemed.
+ */
+export async function getIsTransferCompletedInjective(
+  tokenBridgeAddress: string,
+  signedVAA: Uint8Array,
+  walletPKHash: string,
```

Comment:
I'm hesitant with this as written. We should avoid passing in private keys. Is there not a way to estimate with only the public key like on Terra?

Reply:
If we only have a VAA, then the function needs the private key to sign the transaction.  If the transaction was available, it could be passed in instead of the VAA and as a bonus, the network parameter could just be client.  But, if only the VAA is available, then I don't know how to create a transaction suitable for testing without a private key.

Reply:
If I should exclude this function for now, please advise.

Reply:
implemented the query: https://github.com/certusone/wormhole/pull/1430